cmd/neofs-cli/modules/control/set_netmap_status.go

```diff
@@ -54,6 +57,32 @@ func setNetmapStatus(cmd *cobra.Command, _ []string) {
 		status = control.NetmapStatus_OFFLINE
 	case netmapStatusMaintenance:
 		status = control.NetmapStatus_MAINTENANCE
+
+		common.PrintVerbose("Reading network settings to check allowance of \"%s\" mode...", st)
```

Comment:
Why did you decide to do it from the CLI and not from the storage node?
It is a bit strange to provide 2 endpoints to this command.

Reply:
Because it's good to firstly implement anything on client side, and later consider server-side support. Doing validation on the node side sounds quite good. I guess we don't need `force` flag in this case right?

Reply:
I agree, but in this case we complicate user experience by requiring to provide 2 endpoints.
At the same time `force` flag in control service can allow to move node in "local" maintenance mode even when it is disabled globally with a proper message, and do nothing in other case.